stopstalk_app/lib/classes/user_editorial_class.dart

```diff
@@ -0,0 +1,9 @@
+class UserEditorial {
+  final String editorial_count;
```

Comment:
Refer to this for naming non-const identifiers: [https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html](https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html)